include/gazebo_imu_plugin.h

```diff
@@ -35,23 +35,23 @@
 namespace gazebo {
 //typedef const boost::shared_ptr<const sensor_msgs::msgs::Imu> ImuPtr;
 
-// Default values for use with ADIS16448 IMU
-static constexpr double kDefaultAdisGyroscopeNoiseDensity =
-    2.0 * 35.0 / 3600.0 / 180.0 * M_PI;
-static constexpr double kDefaultAdisGyroscopeRandomWalk =
-    2.0 * 4.0 / 3600.0 / 180.0 * M_PI;
-static constexpr double kDefaultAdisGyroscopeBiasCorrelationTime =
-    1.0e+3;
-static constexpr double kDefaultAdisGyroscopeTurnOnBiasSigma =
-    0.5 / 180.0 * M_PI;
-static constexpr double kDefaultAdisAccelerometerNoiseDensity =
-    2.0 * 2.0e-3;
-static constexpr double kDefaultAdisAccelerometerRandomWalk =
-    2.0 * 3.0e-3;
-static constexpr double kDefaultAdisAccelerometerBiasCorrelationTime =
-    300.0;
-static constexpr double kDefaultAdisAccelerometerTurnOnBiasSigma =
-    20.0e-3 * 9.8;
+// Default values for use with IIM42653 IMU
+static constexpr double kDefaultGyroscopeNoiseDensity =
+    0.0008726646;  // [rad/s/sqrt(Hz)] (0.05 deg/s converted to rad/s)
+static constexpr double kDefaultGyroscopeRandomWalk =
+    0.0;  // [rad/s/s/sqrt(Hz)]
+static constexpr double kDefaultGyroscopeBiasCorrelationTime =
+    1000.0;  // [s]
+static constexpr double kDefaultGyroscopeTurnOnBiasSigma =
+    0.0;  // [rad/s]
+static constexpr double kDefaultAccelerometerNoiseDensity =
+    0.00637;  // [m/s^2/sqrt(Hz)] (0.65 mg-rms converted to m/s^2)
+static constexpr double kDefaultAccelerometerRandomWalk =
+    0.0;  // [m/s^2/s/sqrt(Hz)]
+static constexpr double kDefaultAccelerometerBiasCorrelationTime =
+    300.0;  // [s]
+static constexpr double kDefaultAccelerometerTurnOnBiasSigma =
```

Comment:
Okay for the random walk as it is small, but let's still leave some turn on bias (both gyro and accel), I don't think they're problematic for CI